identity/internal/challenge.go

```diff
@@ -232,7 +206,7 @@ func persistInputsForChallenge(name string, inputs circuits.AtomicQuerySigInputs
 
 func loadClaim(name string) (*ClaimInputsForSigCircuit, error) {
 	homedir, _ := os.UserHomeDir()
-	claimsPath := filepath.Join(homedir, fmt.Sprintf("iden3/%s/received-claims", name))
+	claimsPath := filepath.Join(homedir, fmt.Sprintf("iden3/%s/private/received-claims", name))
```

Comment:
In the JS code you added support for an `IDEN3_WORKDIR` env var. Use that here too? If `workdir` var is used for that (and assuming it includes `iden3` segment as suggested below), this could be simplified to: 
```go
filepath.Join(workdir, name, "private/received-claims")
```